back-end/src/sciler/handler/handler.go

```diff
@@ -348,12 +390,9 @@ func (handler *Handler) HandleEvent(id string) {
 }
 
 func getMapSlice(input interface{}) ([]map[string]interface{}, error) {
-	bytes, err := json.Marshal(input)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
Why is this error check removed?
Is it because the unmarshal will also through an error if there were to occur one?

Reply:
because the confirmation method should not return an error

Reply:
errors are logged in the right place now and not passed around as they should only be logged never panicked